src/main/kotlin/model/ManageRecalls.kt

```diff
@@ -0,0 +1,103 @@
+package model
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.CloudPlatform
+import uk.gov.justice.hmpps.architecture.HMPPSAuth
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.NOMIS
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class ManageRecalls private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var ppcsCaseWorker: Person
+    lateinit var manageRecallsUiContainer: Container
+    lateinit var manageRecallsApiContainer: Container
+    lateinit var dbContainer: Container
+    lateinit var prisonsRegisterContainer: Container
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "Manage Recalls",
+        "The case management system for recalls"
+      )
+
+      dbContainer = system.addContainer(
+        "Manage Recalls API Database",
+        "Database to store recalls caseworking info",
+        "RDS Postgres DB"
+      ).apply {
+        Tags.DATABASE.addTo(this)
+        CloudPlatform.rds.add(this)
+      }
+
+      prisonsRegisterContainer = system.addContainer(
+        "Prisons Register",
+        "Provides a list of prisons",
+        "Kotlin Spring Boot App"
+      ).apply {
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      manageRecallsApiContainer = system.addContainer(
+        "Manage Recalls API",
+        "REST API for the case management of recalls",
+        "Kotlin Spring Boot App"
+      ).apply {
+        setUrl("https://github.com/ministryofjustice/manage-recalls-api")
+        uses(dbContainer, "queries", "JDBC")
+        uses(prisonsRegisterContainer, "validates prisons", "HTTPS")
+        CloudPlatform.kubernetes.add(this)
+        CloudPlatform.s3.add(this)
+        CloudPlatform.elasticache.add(this)
+      }
+
+      manageRecallsUiContainer = system.addContainer(
+        "Manage Recalls Web Application",
+        "Web application for the case management of recalls",
+        "Node Express app"
+      ).apply {
+        setUrl("https://github.com/ministryofjustice/manage-recalls-ui")
+        Tags.WEB_BROWSER.addTo(this)
+        uses(NOMIS.offenderSearch, "searches for prisoners")
```

Comment:
Relationships to things outside this class should be declared in `defineRelationships` (which removes the need to put the new element at the bottom of `MODEL_ITEMS`)

Reply:
ok